test/test_jit.py

```diff
@@ -9090,6 +9090,39 @@ def forward(self, x):
         m.apply(lambda s: s._unpack())
         torch.testing.assert_allclose(m(torch.zeros(3, 4)), torch.ones(3, 4) * 6)
 
+    def test_torch_any(self):
+        def fn(x):
+            return torch.any(x)
+
+        self.checkScript(fn, (torch.randn(3, 4), ))
+        self.checkScript(fn, (torch.empty(3), ))
+        self.checkScript(fn, (torch.empty(1), ))
+        self.checkScript(fn, (torch.ones(3, 4),))
+        self.checkScript(fn, (torch.zeros(5, 7, 1),))
```

Comment:
It looks like torch.any also supports slicing through specified dimension. (torch.any(a, 1)). Could we add a test case for that?

Reply:
Yea, agreed. fixed.

Reply:
cool